lambdas/es/indexer/index.py

```diff
@@ -699,10 +716,9 @@ def handler(event, context):
                     event_["s3"]["object"].get("lastModified") or event_["eventTime"]
                 )
                 # Get two levels of extensions to handle files like .csv.gz
-                path = pathlib.PurePosixPath(key)
-                ext1 = path.suffix
-                ext2 = path.with_suffix('').suffix
-                ext = (ext2 + ext1).lower()
+                ext_next_last, ext_last = get_normalized_extensions(key)
+                compression = get_compression(ext_last)
+                ext = ext_next_last + ext_last
```

Comment:
I can't remember anymore all the details... but if the only reason we support two extensions is to handle `.gz`, then once the compression is factored out, we can just use a single extension, right?

Reply:
I didn't want to change any of the downstream dependencies, for instance we send the full extension, both parts, to ElasticSearch documents via `do_index` and I'm leary of changing that behavior (it would change how certain search results are returned).